scripts/github-status/github-status.rb

```diff
@@ -48,6 +49,11 @@ def get_sha_status(sha)
     get_full_sha_status(sha).state rescue ''
   end
 
+  def get_pr_info(pr)
+    pr_stat = @client.pull_request(@repository, pr)
+    pr_stat.to_attrs
```

Comment:
I totally agree, in fact I had exactly the same idea.  But I am still not even convinced whether we should bloat this script further.  It's the wrong place for such functionality ... I almost closed this PR straight after opening it.